package.json

```diff
@@ -21,6 +21,7 @@
       "pre-commit": "pretty-quick --staged && npm run lint && npm test && npm run typescript"
     }
   },
+  "types": "src/openmrs-esm-primary-navigation.tsx",
```

Comment:
@joeldenning shouldn't this be a declaration file and not `src/openmrs-esm-primary-navigation.tsx`

Reply:
No. See https://github.com/openmrs/openmrs-esm-api/pull/11 for the explanation.